k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/quickstart/cwagent-fluent-bit-quickstart.yaml

```diff
@@ -464,7 +464,7 @@ spec:
     spec:
       containers:
       - name: fluent-bit
-        image: amazon/aws-for-fluent-bit:2.10.0
+        image: amazon/aws-for-fluent-bit:latest
```

Comment:
Personally, I think we need a good way to continuously test/verify the daemonsets before I'd be comfortable switching to use the `latest` tagged image. AFAIK tests for container insights only really happen when ramping up for a release of the agent, so I'd like to have proper integration testing set up prior to approving and merging this in. Whether it happens on this repo or the CloudWatch agent repo, I'm indifferent, but I did cut an [issue](https://github.com/aws/amazon-cloudwatch-agent/issues/442) in the CloudWatch agent repo to at least document this somewhere.